Source/Events.Processing/EventProcessors.cs

```diff
@@ -50,6 +52,12 @@ public class EventProcessors : IEventProcessors
         where TRequest : class
         where TResponse : class
     {
+#if NETCOREAPP3_1
+        if (++_eventProcessoCounter == 100)
```

Comment:
Lets make this warning appear more than once:
```suggestion
        if (++_eventProcessoCounter >= 100)
```

Reply:
😄